packages/server-wallet/src/message-service/types.ts

```diff
@@ -22,6 +22,12 @@ export interface MessageServiceInterface {
    */
   send(message: Message[]): Promise<void>;
 
+  // TODO: Is a url/string too restrictive for addressing a peer?
```

Comment:
I think a `string` will work for everything but it'd be nice if the address type was defined by a  message service implementation so it could be more strongly typed.